sam.c

```diff
@@ -6181,13 +6181,27 @@ int bam_parse_basemod(const bam1_t *b, hts_base_mod_state *state) {
                 cp = cp_end;
                 ms = cp-1;
             } else {
-                while (*cp && *cp != ',' && *cp != ';')
+                while (*cp && isalpha(*cp))
```

Comment:
```suggestion
                while (*cp && isalpha_c(*cp))
```

And the same applies to the `isdigit()` a few lines above.